modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/ConnectionManager.java

```diff
@@ -188,6 +188,8 @@ public void onCacheDestroyed() {
      * Close all connections.
      */
     private void closeConnections() {
+        busyLock.block();
```

Comment:
if we has issue with double invocation the method then I don't understand how it will fix the problem? Second invocation will held here forever